pkg/scheduler/framework/plugins/nodevolumelimits/non_csi_test.go

```diff
@@ -64,6 +64,24 @@ var (
 			HostPath: &v1.HostPathVolumeSource{},
 		},
 	}).Obj()
+	onlyConfigmapAndSecretPod = st.MakePod().Volume(v1.Volume{
+		VolumeSource: v1.VolumeSource{
+			ConfigMap: &v1.ConfigMapVolumeSource{},
+		},
+	}).Volume(v1.Volume{
+		VolumeSource: v1.VolumeSource{
+			Secret: &v1.SecretVolumeSource{},
+		},
+	}).Obj()
+	notOnlyConfigmapAndSecretPod = st.MakePod().PVC("pvcWithDeletedPV").Volume(v1.Volume{
```

Comment:
pvcPodWithConfigmapAndSecret

Reply:
Also add another two cases covering the PerFilter.

Reply:
Already has Ephemeral.